css/custom.css

```diff
@@ -1,252 +1,98 @@
+@import url('../webfonts/Montserrat');
```

Comment:
I don't think this is right

Reply:
Why not? The font is loading for you, isn't it?

Reply:
No I just see the default sans-serif.

The console shows:
> Blocked loading mixed active content “http://vickysteeves.github.io/reprozip/webfonts/Montserrat/”

And that URL is a 404 HTML page: https://vickysteeves.github.io/reprozip/webfonts/Montserrat/